pkg/test/framework/components/echo/config.go

```diff
@@ -49,6 +57,10 @@ type Config struct {
 	// Headless (k8s only) indicates that no ClusterIP should be specified.
 	Headless bool
 
+	// StaticAddress for some echo implementations is an address locally reachable within
+	// the test framework and from the echo Cluster's network.
+	StaticAddresses []string
```

Comment:
Do we ever expect the test code to set this?

Reply:
No. I wanted to put this in a better spot, but it was difficult while avoiding circular imports.